pkg/api/k8s_proxy.go

```diff
@@ -0,0 +1,61 @@
+package api
+
+import (
+	"crypto/tls"
+	"net"
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"strconv"
+	"time"
+
+	"github.com/grafana/grafana/pkg/infra/appcontext"
+	"github.com/grafana/grafana/pkg/infra/log"
+	contextmodel "github.com/grafana/grafana/pkg/services/contexthandler/model"
+	"github.com/grafana/grafana/pkg/util"
+	"github.com/grafana/grafana/pkg/util/proxyutil"
+	"github.com/grafana/grafana/pkg/web"
+)
+
+var k8sProxyTransport = &http.Transport{
+	Proxy: http.ProxyFromEnvironment,
+	DialContext: (&net.Dialer{
+		Timeout:   30 * time.Second,
+		KeepAlive: 30 * time.Second,
+	}).DialContext,
+	TLSHandshakeTimeout: 10 * time.Second,
+	TLSClientConfig: &tls.Config{
+		InsecureSkipVerify: true,
```

Comment:
don't we need to set `Certificates` or `GetClientCertificate` here to provide the client cert we'll send to k8s?

Reply:
🤔 yeah, i think some other config settings on that branch might be hiding this error. i'll look into it